core/src/main/resources/error/error-classes.json

```diff
@@ -70,6 +70,11 @@
       "Another instance of this query was just started by a concurrent session."
     ]
   },
+  "CONVERSION_INVALID_INPUT" : {
```

Comment:
This is similar to `CAST_INVALID_INPUT`, it seems. Can you re-use the existing one?

Reply:
I think they are semantically different enough to require separate error class. `TO_BINARY` is string encoding to type conversion, while `CAST` is type to type conversion.

Reply:
ok, I see.

Reply:
shall we make the name more explicit? `TO_BINARY_INVALID_INPUT`

Reply:
I think it will be helpful to generalize this for other conversions, e.g. as mentioned below - `encode`, `unhex`, `unbase64`, and vice versa.